src/python/grpcio/grpc/_channel.py

```diff
@@ -1340,7 +1484,9 @@ def _deliveries(state):
     return callbacks_needing_update
 
 
-def _deliver(state, initial_connectivity, initial_callbacks):
+def _deliver(state: _ChannelConnectivityState,
+             initial_connectivity: grpc.ChannelConnectivity,
+             initial_callbacks: Sequence[Callable]) -> None:
```

Comment:
`Sequence[Callable[[grpc.ChannelConnectivity], None]]`

Reply:
it should be `List`, changed to `List[Callable[[grpc.ChannelConnectivity], None]]`